tests/test_compliance/dataframe_helper.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+import pytest
+import yaml
+from linkml_runtime.utils.compile_python import compile_python
+
+_MIN_POLARS_VERSION = "1.29.0"
+
+logger = logging.getLogger(__name__)
+
+
+def apply_skip_list(skip_value: str, skip_list: list[str]) -> None:
+    """Skip tests that match a string (for example schema name)"""
+    for n in skip_list:
+        if skip_value.startswith(n):
+            pytest.skip(reason=f"Skipping test due to match on {n}")
+
+
+_SKIP_LIST = [
+    "test_inlined-INLTrue_IALFalse_MVTrue_FKFalse",
+    "test_inlined-INLTrue_IALTrue_MVTrue_FKFalse",
+    "test_jsonpointer",
+    "test_inlined_unique_keys",
+    "test_nested_key",
+    "test_date_types",
+    "test_array",
+    "test_slot_any_of",
+    "test_membership",
+    "test_class_boolean",
+    "test_non_standard",
+    "test_equals_string",
+    "test_value_presence",
+    "test_cardinality-MVTrue_REQFalse",
+    "test_cardinality-MVTrue_REQTrue",
+    "test_cardinality-ClassNameEQ_C__SlotNameEQ_sSPACE1__TypeNameEQ_tSPACE1",
+    "test_cardinality-ClassNameEQ_C__SlotNameEQ_1s__TypeNameEQ_T1",
+    "test_cardinality-ClassNameEQ_C__SlotNameEQ_1s__TypeNameEQ_T1",
+]
```

Comment:
Are we skipping these because they are not implemented or unnecessary? Will we be dealing with these later somehow?